iiblib/iib_client.py

```diff
@@ -435,6 +436,48 @@ def create_empty_index(
             return resp.json()
         return CreateEmptyIndexModel.from_dict(resp.json())
 
+    @retry(
+        retry=retry_if_exception_type(IIBRecoverableError),
+        stop=stop_after_attempt(STATE_REASON_RETRY),
+    )
+    def recursive_related_bundles(
```

Comment:
I am wondering if we really need this? Pub is not using recursive_related_bundles. 
So in theory we just need the model for passing iib-api-tests, right?

There were some discussions if we should implement every API or not. Did we agreed to maintain only APIs related to Pub? 

I am asking to reduce the amount of work. If we introduce this new method we should cover it in iib-api-test (integration) as well.

@yashvardhannanavati @chandwanitulsi

Reply:
It seems that MergeIndexImageModel, is done in that way.

Reply:
ok, so I would suggest to keep it that way.